common/dna_data/wc_human_dna.txt

```diff
@@ -2699,4 +2699,608 @@ aiden_perenolde_dna = { # by ercarp
+ 		gene_bs_nose_ridge_width={ "nose_ridge_width_neg" 5 "nose_ridge_width_neg" 73 }
+ 		gene_bs_nose_size={ "nose_size_pos" 115 "nose_size_pos" 31 }
+ 		gene_bs_nose_tip_angle={ "nose_tip_angle_pos" 9 "nose_tip_angle_neg" 9 }
+ 		gene_bs_nose_tip_forward={ "nose_tip_forward_neg" 55 "nose_tip_forward_pos" 22 }
+ 		gene_bs_nose_tip_width={ "nose_tip_width_neg" 101 "nose_tip_width_neg" 41 }
+ 		face_detail_cheek_def={ "cheek_def_01" 153 "cheek_def_02" 21 }
+ 		face_detail_cheek_fat={ "cheek_fat_04_pos" 58 "cheek_fat_01_neg" 37 }
+ 		face_detail_chin_cleft={ "chin_cleft" 137 "chin_dimple" 18 }
+ 		face_detail_chin_def={ "chin_def" 255 "chin_def" 10 }
+ 		face_detail_eye_lower_lid_def={ "eye_lower_lid_def" 116 "eye_lower_lid_def" 19 }
+ 		face_detail_eye_socket={ "eye_socket_01" 149 "eye_socket_03" 125 }
+ 		face_detail_nasolabial={ "nasolabial_02" 157 "nasolabial_03" 5 }
+ 		face_detail_nose_ridge_def={ "nose_ridge_def_pos" 0 "nose_ridge_def_pos" 7 }
+ 		face_detail_nose_tip_def={ "nose_tip_def" 46 "nose_tip_def" 18 }
+ 		face_detail_temple_def={ "temple_def" 194 "temple_def" 195 }
+ 		expression_brow_wrinkles={ "brow_wrinkles_01" 27 "brow_wrinkles_01" 27 }
+ 		expression_eye_wrinkles={ "eye_wrinkles_01" 2 "eye_wrinkles_01" 2 }
+ 		expression_forehead_wrinkles={ "forehead_wrinkles_02" 177 "forehead_wrinkles_02" 177 }
+ 		expression_other={ "cheek_wrinkles_both_01" 0 "cheek_wrinkles_both_01" 0 }
+ 		complexion={ "complexion_2" 112 "complexion_7" 170 }
+ 		gene_height={ "normal_height" 120 "normal_height" 128 }
+ 		gene_bs_body_type={ "body_fat_head_fat_medium" 147 "body_fat_head_fat_medium" 147 }
+ 		gene_bs_body_shape={ "body_shape_average_clothed" 153 "body_shape_hourglass_full" 0 }
+ 		gene_bs_bust={ "bust_clothes" 106 "bust_shape_3_half" 93 }
+ 		gene_age={ "old_2" 107 "old_2" 107 }
+ 		gene_eyebrows_shape={ "avg_spacing_lower_thickness" 215 "far_spacing_low_thickness" 161 }
+ 		gene_eyebrows_fullness={ "layer_2_lower_thickness" 216 "layer_2_low_thickness" 129 }
+ 		gene_body_hair={ "body_hair_avg" 177 "body_hair_avg" 177 }
+ 		gene_hair_type={ "hair_wavy" 124 "hair_wavy" 124 }
+ 		gene_baldness={ "no_baldness" 127 "no_baldness" 127 }
+ 		eye_accessory={ "normal_eyes" 39 "normal_eyes" 39 }
+ 		teeth_accessory={ "normal_teeth" 0 "normal_teeth" 0 }
+ 		eyelashes_accessory={ "normal_eyelashes" 103 "normal_eyelashes" 103 }
+ 		gene_bs_ear_lenght={ "floppy_ear_length" 0 "floppy_ear_length" 0 }
+ 		special_eyebrows={ "no_eyebrows" 72 "no_eyebrows" 72 }
+ 		special_eyes={ "no_special_eyes" 0 "no_special_eyes" 0 }
+ 		gene_bs_mouth_tusks={ "mouth_fangs" 0 "mouth_fangs" 0 }
+ 		static_tusks={ "no_static_tusks" 0 "no_static_tusks" 0 }
+ 		gene_bs_ear_tip={ "ear_tip_pos" 0 "ear_tip_pos" 0 }
+ 		gene_bs_fingers={ "fingers_troll" 0 "fingers_troll" 0 }
+ 		gene_skin_value={ "skin_value_pos" 0 "skin_value_neg" 0 }
+ 		gene_race={ "creature_human" 61 "creature_human" 61 }
+        }
+    }
+	enabled=yes
+}
+
+alonsus_faol_dna = {
```

Comment:
What did you use for a reference? On my arts he has a pointy nose and doesn't have chubby cheeks.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/123140343-3cff8c80-d468-11eb-8bff-90ca96f2fba9.png)

</details>

Reply:
it is said in wowpedia that he is similar to santa clause, short and stout

Reply:
i made him more slim, i havent just updated these files